src/tools/compiletest/src/directive-list.rs

```diff
@@ -48,6 +48,7 @@ const KNOWN_DIRECTIVE_NAMES: &[&str] = &[
     "ignore-coverage-run",
     "ignore-cross-compile",
     "ignore-eabi",
+    "ignore-elf",
```

Comment:
Suggestion: could you add `{ignore,only}-elf` to the compiletest directive listing in rustc-dev-guide so people can find out about it?